### DISCUSSION TITLE
Completely new for me

### DISCUSSION THREAD

**Discussion by Midgas56** [sqlitebrowser/sqlitebrowser#3630] [General]
Hi guys, I'm discovering SQLite and I would want to know how to modify an existing table succesfully. By modifying I mean put some columns of some tables in another table so I would be more comfortable to make changes on the DB. Thanks and sorry my approximative english.

**Comment by justinclift** [comment]
Ahhh.  That's the kind of thing where you'd probably be wanting to ask on the SQLite Forum:

https://sqlite.org/forum/forum

They're pretty friendly, and tend to respond fairly quickly, so it's a good avenue for people wanting to ask questions. :smile:

**Reply by Midgas56** [reply]
Merci !

**Comment by chrisjlocke** [comment]
> put some columns of some tables in another table so I would be more comfortable to make changes on the DB

Do you mean copy the data or just the fields (columns)?  I'm not too clear on what you're trying to achieve.

Do you mean create a copy of a table to experiment on?

The easiest method is to create a copy of the table and then copy data into this table.

![image](https://github.com/sqlitebrowser/sqlitebrowser/assets/7872218/350faad9-6770-4154-8de2-ad88df1b5ae9)

Right-click the table and select 'Copy create statement'

![image](https://github.com/sqlitebrowser/sqlitebrowser/assets/7872218/0013d1eb-ea1d-4ef8-bbb5-585d1e748bda)

switch to the 'execute sql' tab and paste in the code, but change the table name.

Copy the existing data by typing

```
insert into b
select *
from a
```

where 'b' is the new table, and 'a' is the original table.

**Reply by chrisjlocke** [reply]
Did you get any further with this?

**Reply by Midgas56** [reply]
J'ai besoin d'avoir la copie d'une partie d'une table pour la transposer sur une autre afin de me permettre de mieux filtrer les informations de ma Database. Je ne sais pas du tout si je suis bien clair.

**Reply by Midgas56** [reply]
Je vais essayer de suivre vos instructions. Merci bien !
